charcounter/logging_config.py

```diff
@@ -0,0 +1,17 @@
+import logging
+
+
+def _logger_factory() -> logging.Logger:
+    logger = logging.getLogger(__name__)
+    logger.setLevel("DEBUG")
+    formatter = logging.Formatter("{asctime} - {levelname} - {message}", style="{", datefmt="%Y-%m-%d %H:%M")
+
+    console_handler = logging.StreamHandler()
+    console_handler.setLevel("DEBUG")
+    console_handler.setFormatter(formatter)
+    logger.addHandler(console_handler)
+
+    return logger
+
+
+logger: logging.Logger = _logger_factory()
```

Comment:
`: logging.Logger` in this case is redundant because you have a very specific return type annotation of your`def _logger_factory() -> logging.Logger:` . Python sees -> logging.Logger.